coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
+    """
+    Escapes the given string with `[c]` pattern.
+
+    :param string: String that is to be escaped with `[ ]`.
+    :return:       Escaped string in which all the special glob characters
+                   `()[]|?*` are escaped.
+    """
+    output = ""
+    special_chars = "()[]|?*"
```

Comment:
If these are all the special chars in Globbing that we handle, we should make it a variable in `Constants.py` or a constant variable in this module (so that other modules can use it too ?)

something like `Globbing.SPECIAL_CHARS` ?